next-prayer

```diff
@@ -100,7 +100,7 @@ hybrid() {
 [[ ${adhan_time} == "yes" ]] && notify-send -t 60000 "Next Prayer" "NOW It's the time for ${next_prayer}"
 
 case "$1" in
-	"--prev"|"-p") echo "🕌 ${prev_prayer} $(format ${prev_prayer_time})" ;;
+	"--previous"|"-p") echo "🕌 ${prev_prayer} $(format ${prev_prayer_time})" ;;
```

Comment:
Instead of fixing all `prev` related typos in different files, replace `--previous` flag in [README.md](https://github.com/AbdeltwabMF/next-prayer/blob/main/README.md) with `--prev`.